frontend/app/routes.json

```
@@ -258,6 +258,14 @@
                           "en": "/:lang/apply/:id/adult-child/confirmation",
                           "fr": "/:lang/demander/:id/adulte-enfant/confirmation"
                         }
+                      },
+                      {
+                        "id": "$lang+/_public+/apply+/$id+/adult-child/apply-for-yourself",
+                        "file": "routes/$lang+/_public+/apply+/$id+/adult-child/apply-for-yourself.tsx",
+                        "paths": {
+                          "en": "/:lang/apply/:id/adult-child/apply-for-yourself",
+                          "fr": "/:lang/demander/:id/adulte-enfant/postulez-pour-vous-meme"
```

Based on [URL model](https://design.canada.ca/architecture/organizing-content.html#toc3), we should remove `for` from the url. Tename the route file as well.

> avoid superfluous words such as “to”, “the”, “a”, “an”, “by”, “for” (for example, “/apply-student-loan” instead of “/how-to-apply-for-a-canadian-student-loan”

```suggestion
                        "id": "$lang+/_public+/apply+/$id+/adult-child/apply-yourself",
                        "file": "routes/$lang+/_public+/apply+/$id+/adult-child/apply-yourself.tsx",
                        "paths": {
                          "en": "/:lang/apply/:id/adult-child/apply-yourself",
                          "fr": "/:lang/demander/:id/adulte-enfant/postulez-vous-meme"
```